scalafix-tests/input/src/main/scala-2/NoSemanticComment.scala

```diff
@@ -0,0 +1,4 @@
+package test
+
+// This has no testkit semantic comment and should be gracefully ignored by the testkit
+object NoSemanticComment
```

Comment:
Nit: newline

Reply:
FTR, such files should actually raise an error per the current implementation, so I removed it